in_toto/verifylib.py

```diff
@@ -1315,6 +1314,10 @@ def in_toto_verify(layout, layout_key_dict):
   # container (Metablock) that also carries the signatures
   layout = layout.signed
 
+  log.info("Begin printing layout metadata")
+  log.info(layout.display())
+  log.info("End printing layout metadata")
```

Comment:
Please don't call you function here or in `runlib`.

Reply:
I've removed these function calls from both `verifylib` and `runlib` in the latest commits.